Makefile

```diff
@@ -0,0 +1,30 @@
+# Contains helpful targets for building and running the project.
+
+all: build
+
+.PHONY: build
+build: run
+
+run: run.c
+	gcc -O3 -o run run.c -lm
+
+.PHONY: run-inference
+run-inference: .venv run out/model.bin
```

Comment:
I'd also probably add a `run-fully-optimized` which takes advantage of as many compiler flags as possible with something like:

`run-native-optimized: gcc -Ofast -funsafe-math-optimizations -march=native -flto`